tests/TestGalaxyObjects.py

```diff
@@ -617,6 +627,29 @@ def test_update(self):
         self.assertEqual(self.hist.annotation, new_annotation)
         self.assertEqual(self.hist.tags, new_tags)
 
```

Comment:
You need to decorate the `test_new_dataset_collection`, `test_delete_dataset_collection` and `test_run_workflow_with_dataset_collection` tests with 
```
@test_util.skip_unless_galaxy('release_14.06')
```
because dataset collections where not available before Galaxy 14.06.